great-idea/index.html

```diff
@@ -15,9 +15,76 @@
 </head>
 
   <body>
+    <div class="max-width">
+      <div class = "nav-bar-div">
+        <nav class = "nav-bar">
+          <a href="#">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        
+        <div class="logo">
+        <img src="img/logo.png" alt="Great Idea! Company logo.">
+      </div>
+    </nav>
+      </div>
+  <header>
+    <section class = "firstSection">
+      <div class ="title">
+    <h1>Innovation <br>On <br>Demand</h1>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
+    <button class="get-started">Get Started</button>
+  </div>
+    
+  <div class="middle-img">
+    <img class="top-img" src="img/header-img.png" alt="Image of a code snippet.">
+  </div>
+  </section>
+  </header>
+  <section class = "top-section">
+  <div>
+    <h3>Features</h3>
```

Comment:
consider using the "format" feature in visual studio code to fix the indenting. that will make the code easier to read.